src/app/workflow/workflow.component.html

```diff
@@ -132,7 +132,7 @@ <h3>
         <app-tool-tab [selectedVersion]="selectedVersion"></app-tool-tab>
       </tab>
       <tab id="dagTab" [disabled]="isStub()" heading="DAG" *ngIf="validVersions" (select)="setTab('DAG')">
-        <app-dag *ngIf="checkMode('DAG')" [id]="workflow?.id" [validVersions]="validVersions" [defaultVersion]="defaultVersion" [selectedVersion]="selectedVersion"></app-dag>
+        <app-dag *ngIf="checkMode('DAG')" [id]="workflow?.id" [validVersions]="validVersions" [defaultVersion]="defaultVersion" [selectedVersion]="selectedVersion" [theWorkflow]="workflow"></app-dag>
```

Comment:
If passing in the entire workflow, see if [id] can be removed

Reply:
Good catch, thanks.